src/js/components/common/LabelIcon.react.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import React from 'react';
+
+interface Props {
+  children: string;
+  required: ?boolean;
+  valid: ?boolean;
+}
+
+const LabelIcon = ({ children, required = false, valid = false }: Props) => {
```

Comment:
Can we make this component more extensible by sending the icon and tooltip as a prop instead of the required/valid, so we just define here how it would look and behave like and not other logic that depends on the props? We can set default values for icon and tooltip to the optional here if needed only